### ISSUE TITLE
Strava gets find everytime

### ISSUE THREAD

**Issue by Pietagorh** [sherlock-project/sherlock#1632]
## Checklist

- [x] I'm reporting a bug in Sherlock's functionality
- [x] The bug I'm reporting is not a false positive or a false negative
- [x] I've verified that I'm running the latest version of Sherlock
- [x] I've checked for similar bug reports including closed ones
- [x] I've checked for pull requests that attempt to fix this bug

## Description

Strava gets find on every request
I checked ~20 usernames, half of wich are just me typing random things, and got strava everytime, even for my pseudo even though I ever used any sport app
I think it comes from the change of Strava's website (on the 3rd of November according to the wayback machine)
```C:\Users\Administrateur>python sherlock/sherlock sherlocksherlocksherlocksherlocksherlocksherlocksherlock
[*] Checking username sherlocksherlocksherlocksherlocksherlocksherlocksherlock on:

[+] Strava: https://www.strava.com/athletes/sherlocksherlocksherlocksherlocksherlocksherlocksherlock
[+] Twitch: https://www.twitch.tv/sherlocksherlocksherlocksherlocksherlocksherlocksherlock

[*] Results: 2

[!] End:  The processing has been finished.

C:\Users\Administrateur>python sherlock/sherlock aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa
[*] Checking username aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa on:


C:\Users\Administrateur>python sherlock/sherlock aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa
[*] Checking username aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa on:

[+] Instagram: https://www.instagram.com/aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa
[+] Slashdot: https://slashdot.org/~aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa
[+] Strava: https://www.strava.com/athletes/aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa
[+] iMGSRC.RU: https://imgsrc.ru/main/user.php?user=aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababa

[*] Results: 4

[!] End:  The processing has been finished.

C:\Users\Administrateur>python sherlock/sherlock aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababaekzhjfkehkfbzke
[*] Checking username aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababaekzhjfkehkfbzke on:

[+] Slashdot: https://slashdot.org/~aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababaekzhjfkehkfbzke
[+] Strava: https://www.strava.com/athletes/aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababaekzhjfkehkfbzke
[+] iMGSRC.RU: https://imgsrc.ru/main/user.php?user=aaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbabababababababaekzhjfkehkfbzke

[*] Results: 3

[!] End:  The processing has been finished.

C:\Users\Administrateur>python sherlock/sherlock Pietagorh
[*] Checking username Pietagorh on:

[+] Crowdin: https://crowdin.com/profile/Pietagorh
[+] Facebook: https://www.facebook.com/Pietagorh
[+] GitHub: https://www.github.com/Pietagorh
[+] Minecraft: https://api.mojang.com/users/profiles/minecraft/Pietagorh
[+] MyAnimeList: https://myanimelist.net/profile/Pietagorh
[+] Pastebin: https://pastebin.com/u/Pietagorh
[+] Periscope: https://www.periscope.tv/Pietagorh/
[+] Quizlet: https://quizlet.com/Pietagorh
[+] Reddit: https://www.reddit.com/user/Pietagorh
[+] Sketchfab: https://sketchfab.com/Pietagorh
[+] Strava: https://www.strava.com/athletes/Pietagorh
[+] Trello: https://trello.com/Pietagorh
[+] Twitch: https://www.twitch.tv/Pietagorh
[+] Twitter: https://twitter.com/Pietagorh

[*] Results: 15

[!] End:  The processing has been finished.```

**Comment by lightlybakedman** [comment]
I'm having the same issues with facebook and zoomit.ir.

**Comment by TheAlmightyDada** [comment]
I'm having the same issue with Coil

**Comment by Pietagorh** [comment]
> I'm having the same issues with facebook and zoomit.ir.

For facebook, could be that an Instagram account with that name exists, cause it's weirdly linked or something

**Comment by sdushantha** [comment]
Issue has been fixed in #1675
@Pietagorh @lightlybakedman
